### DISCUSSION TITLE
Remote Student - Rejected

### DISCUSSION THREAD

**Discussion by PurpleCharger** [community/community#59044] [GitHub Education]
### Hi everyone,

Hi all,

After reading through the recent posts here, I'm going to add to the seemingly endless issues with GitHubs student verification process (no PDFs? What the...?).

I've been rejected because my location is not where the campus is. That's true and the document I submitted clearly lists my learning location as "online" as opposed to one of the physical campus locations.

There does not seem to be a way to submit a response to the auto rejection or escalate it in any way.

How do I request an actual human read my documentation?

It took 5 attempts to get this close.

Thanks.

**Comment by yourdevujjawal** [comment]
Same issue is with me. I am enrolled in Online degree and Github is giving me error that it can't verify because I am not near a campus.

**Comment by samus-aran** [comment]
Hi @PurpleCharger 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Does your school have any other academic emails?
